VideoLocker/src/main/java/org/edx/mobile/module/prefs/PrefManager.java

```diff
@@ -404,4 +412,11 @@ public void setLastCourseStructureFetch(String courseId, long timestamp) {
         public static final String BACKEND_FACEBOOK = "facebook";
         public static final String BACKEND_GOOGLE = "google-oauth2";
     }
+
+    public static void nukeSharedPreferences() {
```

Comment:
Why not just iterate through the 'shared_prefs' directory and delete all the pref files, instead of just emptying them?